src/rulesets/mergers/__tests__/rules.jest.test.ts

```diff
@@ -146,18 +118,6 @@ describe('Ruleset rules merging', () => {
     expect(rules).toHaveProperty('test.severity', -1);
   });
 
-  it('supports array-ish syntax', () => {
```

Comment:
This was one was a dupe. See https://github.com/stoplightio/spectral/blob/6661202220798ce5139c16bcf887fbdf4721730f/src/rulesets/mergers/__tests__/rules.jest.test.ts#L109